### DISCUSSION TITLE
Why so long without new releases?

### DISCUSSION THREAD

**Discussion by centralhardware** [photoprism/photoprism#3347] [Q&A] [answered]
?

**Comment by lastzero** [comment, marked as answer]
This is because we were busy adding many enhancements and new features:

- https://docs.photoprism.app/release-notes/#development-preview

We just need a little more time to complete the work on the Stripe integration, as publicly announced on Twitter and Mastodon:

https://twitter.com/photoprism_app/status/1648712094160764928

Thank you for your patience! ❤️
